azure-pipelines.yml

```diff
@@ -278,6 +278,10 @@ jobs:
         --target_pattern_file="${TARGET_PATTERN_FILE}"
     displayName: Build & test SW
   - template: ci/publish-bazel-test-results.yml
+  - bash: ci/scripts/check-unrunnable-tests.sh
```

Comment:
Maybe just drop a comment here why this is not run during the lint job?

Reply:
done